mappings/charging_station.yml

```diff
@@ -1,18 +1,17 @@
 name: Stations de recharge de véhicules électriques
 overpass: nwr["amenity"="charging_station"](area.searchArea);
 mapping:
-  covered: covered|bool
   opening_hours: opening_hours
   parking_fee: "parking:fee"
-  start_date: start_date
   owner: owner
   operator: operator
   website: website
-  chademo: "socket:chademo|bool"
+  capacity: capacity
+  chademo: "socket:chademo"
```

Comment:
Oh, these are number of sockets, right? Then, maybe "socket:chademo|int" for the same reason?

Reply:
fixed